astropy/table/table.py

```diff
@@ -1903,6 +1911,48 @@ def remove_rows(self, row_specifier):
         if hasattr(self, '_groups'):
             del self._groups
 
+    def remove_masked_rows(self):
+        '''
+        Remove masked rows from a table.
+
+        Examples
+        --------
+        Create a table with three columns 'a', 'b' and 'c'::
+
+            >>> weather_data = """
+            ...      day,temp,type
+            ...      ,35,rainy
+            ...      Wed,31,snowy
+            ...      Thu,25,snowy
+            ...      Sun,1.1,
+            ...      """
+
+            you would read it like this: t = ascii.read(weather_data)
```

Comment:
why do you suggest to use the `ascii` API rather than `Table`?

Reply:
I was using `ascii` because that's how this issue was started. I didn't know before you pointed out how to make a Table with format specified. Will be solved in the next commit